charts/renku/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: '1.0'
 description: A Helm chart for the Renku platform
 name: renku
 icon: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/raw/master/renku_sphinx_theme/static/favicon.png
-version: 0.7.2
+version: 0.7.2-3034fe0
```

Comment:
We normally update this only when making releases, I dont think is necessary to include this change in the PR.

Reply:
This is the result of using `chartpress`, I'm including it in the PR so the tests can run correctly (as they need to pick a new image).
Once the review is done I'll reset them before merging.